cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -336,15 +423,8 @@ def test_terminate(provider, testing_instance, verify_vm_running, soft_assert, a
     msg = "{} for 1 VM and Instance from the {} Database".format(msg_part, appliance.product_name)
     view = appliance.browser.create_view(BaseLoggedInPage)
     view.flash.assert_success_message(msg)
-    terminated_states = (
-        testing_instance.STATE_TERMINATED, testing_instance.STATE_ARCHIVED,
-        testing_instance.STATE_UNKNOWN
-    )
-    soft_assert(
-        testing_instance.wait_for_instance_state_change(
-            desired_state=terminated_states, timeout=1200
-        )
-    )
+    wait_for_instance_state(soft_assert, provider, testing_instance, state="terminated")
```

Comment:
**REQUIRED:** Previously this would have also checked for a state of archived and unknown too. Are we sure we want to drop that? Just checking as it is a change.

Reply:
@psav calling `wait_for_instance_state` with `state="terminated"` still checks for those 3 states to appear on the UI, and does not check the provider backend (which appears to be what these tests were doing here). See https://github.com/bsquizz/integration_tests/blob/f63b29616d94bed28063f0d1f1cf07fc9a059587/cfme/tests/cloud/test_instance_power_control.py#L173-L195